WordPress/Tests/WP/I18nUnitTest.4.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+/*
+ * Test sniffing for strings wrapped in HTML.
+ */
+// phpcs:set WordPress.WP.I18n text_domain[] my-slug
+
+ __( '<address>123 Fake Street</address>', 'my-slug' ); // Bad, string shouldn't be wrapped in HTML.
+ __( 'I live at <address>123 Fake Street</address>', 'my-slug' ); // Okay, no consensus on HTML tags withing strings.
```

Comment:
I would like to see some more unit tests. This feels very minimal for a new feature.